modules/apps/roles/roles-admin-web/src/main/resources/META-INF/resources/edit_role_assignments.jsp

```diff
@@ -56,12 +56,44 @@ renderResponse.setTitle(role.getTitle(locale));
 	navigationItems="<%= roleDisplayContext.getRoleAssignmentsNavigationItems(portletURL) %>"
 />
 
-<clay:management-toolbar-v2
+<portlet:actionURL name="editRoleAssignments" var="editRoleAssignmentsURL">
+	<portlet:param name="mvcPath" value="/edit_role_assignments.jsp" />
+	<portlet:param name="tabs1" value="assignees" />
+</portlet:actionURL>
+
+<portlet:renderURL var="selectAssigneesURL" windowState="<%= LiferayWindowState.POP_UP.toString() %>">
+	<portlet:param name="mvcPath" value="/select_assignees.jsp" />
+	<portlet:param name="tabs2" value="<%= tabs2 %>" />
+	<portlet:param name="tabs3" value="available" />
+	<portlet:param name="redirect" value="<%= currentURL %>" />
+	<portlet:param name="roleId" value="<%= String.valueOf(roleId) %>" />
+	<portlet:param name="displayStyle" value="<%= displayStyle %>" />
+</portlet:renderURL>
+
+<%
+HashMap<String, Object> additionalProps = HashMapBuilder.<String, Object>put(
+	"assigneeType", HtmlUtil.escapeJS(tabs2)
+).put(
+	"editRoleAssignmentsURL", editRoleAssignmentsURL.toString()
+).put(
+	"modalSegmentState", RolesAdminWebKeys.MODAL_SEGMENT_STATE
+).put(
+	"portletURL", portletURL.toString()
+).put(
+	"roleName", role.getName()
+).put(
+	"selectAssigneesURL", selectAssigneesURL.toString()
+).build();
+%>
+
+<clay:management-toolbar
 	actionDropdownItems="<%= editRoleAssignmentsManagementToolbarDisplayContext.getActionDropdownItems() %>"
+	additionalProps="<%= additionalProps %>"
 	clearResultsURL="<%= editRoleAssignmentsManagementToolbarDisplayContext.getClearResultsURL() %>"
-	componentId="editRoleAssignmentsManagementToolbar"
+	creationMenu="<%= editRoleAssignmentsManagementToolbarDisplayContext.getCreationMenu() %>"
```

Comment:
It's a bit weird that we are adding `creationMenu` here. How did this work before? Maybe it was intentional that there is no menu and we fallback to `onClick` on a button.

Reply:
Hey @markocikos good catch ... I think it's a "left over" something I added at one point that isn't needed.
Let me check and confirm that - and I'll update this PR

Reply:
Updated in [https://github.com/liferay-frontend/liferay-portal/pull/835/commits/4414fb3b9bfc1ae71b5b6686df5a01126568cefb](https://github.com/liferay-frontend/liferay-portal/pull/835/commits/4414fb3b9bfc1ae71b5b6686df5a01126568cefb)